login/logout.js

```diff
@@ -0,0 +1,12 @@
+const core = require('@actions/core')
+const exec = require('@actions/exec');
+
+const telepresenceLogout = async function(){
+    try {
+        await exec.exec('telepresence', ['logout']);
```

Comment:
now that TP has released a new version I've been doing some tests with the emoji service and looks like when using v2.6.0 in the CI (which for now doesn't work) and then using v2.5.8 in another version the cluster (or traffic agent) expects a client at version 2.6.0 due to the first run. My question is: Should we uninstall TP after we run integration tests so we left the cluster as it originally was?

Reply:
please disregard - as we spoke in our sync up. This is due to the side effects of installing TP using the `install` command